src/main/java/tests/OperationHandlersTest.java

```diff
@@ -0,0 +1,65 @@
+package tests;
+
+import core.basesyntax.*;
```

Comment:
It's better to avoid wildcard imports as they can lead to importing unnecessary classes and can make the code less readable. Import only the classes that are needed.